tests/java/Makefile

```diff
@@ -1,8 +1,13 @@
 
 KAFKA_JARS?=$(KAFKA_PATH)/libs
 
+all: *.class
```

Comment:
Does that right hand side really work? That looks like it will match *.class files, but the intention is probably to match ..class make targets.

I suggest you do:
```
CLASSES=Murmur2Cli.class TransactionProducerCli.class


all: $(CLASSES)
```

Reply:
I last used make, poorly, about 15 years ago. I assume you're right.